ocs_ci/helpers/helpers.py

```diff
@@ -505,6 +505,7 @@ def create_storage_class(
     provisioner=None,
     rbd_thick_provision=False,
     encrypted=False,
+    encryptionkmsid=None,
```

Comment:
Can we make this more readable with underscores? `encryption_kms_id`?

Reply:
done